tf_files/aws/modules/elasticsearch-alarms/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "slack_webhook" {
+  default = ""
+}
+
+variable "ebs_volume_size" {
+  default = ""
+}
+
+variable "alarm_threshold" {
+  default = "85"
+}
+
+variable "vpc_name" {
+  default = ""
+}
+
+variable "es_domain_name" {
+  default = ""
+}
+
+variable "client_id" {
```

Comment:
I think you want to get rid of this variable if you are not using it anymore

Reply:
Good call I missed that one.